### ISSUE TITLE
loss in the code is not the same as in the paper

### ISSUE THREAD

**Issue by kyrawu327** [yang-yk/NP-RepMet#4]
Hi, thank for the nice work and the idea of restoring negative information is quite inspiring.
However, I find that the loss in the code is not the same as in the paper, am I missing something? or the code is further optimized?

Looking forward to your kind reply.

**Comment by yang-yk** [comment]
> Hi, thank for the nice work and the idea of restoring negative information is quite inspiring.
> However, I find that the loss in the code is not the same as in the paper, am I missing something? or the code is further optimized?
> 
> Looking forward to your kind reply.

Yes. The loss is further optimized with the cosine distance. Thanks！
